src/components/App/index.js

```diff
@@ -24,7 +24,26 @@ export default function App() {
         </MenuItem>
       </TopBar>
       <Workspace id="main-content" tabIndex={-1}>
-        <h1>Color Palette</h1>
+        <h1>Color Palette Contrast Checker</h1>
+        <p>
+          Selecting high contrast color combinations is critical to deliver
+          accessible content. The W3C&apos;s Web Content Accessibility
+          Guidelines or WCAG, suggest that foreground text over background
+          colors meet a minimum contrast ratio. Adhering to these guidelines
+          enables all users, including people vision impairments, to enjoy your
+          content.
+        </p>
+        <h2>How to use</h2>
+        <p>
+          Import the hex or RGB color values that comprise your current color
+          palette to see where they stand against WCAG 2.0 AA color contrast
```

Comment:
2.1?

Reply:
👍 this is just copy pasta from ryan's document. I will update this to 2.1